lightkurve/tests/test_targetpixelfile.py

```diff
@@ -378,18 +378,28 @@ def test_tess_simulation():
     assert np.isnan(tpf.time).sum() == 0
```

Comment:
If we have this, do we need the "TESS_SIM" data set and `test_tess_simulation()` function?  Can you delete these two things?